docs/user-guide/src/bmo/provisioning.md

```diff
@@ -0,0 +1,127 @@
+# Provisioning and Deprovisioning
+
+The most fundamental feature of Metal3 BareMetal Operator is provisioning of
```

Comment:
```suggestion
The most fundamental feature of Metal3 Bare Metal Operator is provisioning of
```
Once again, we should standardize the spelling of our components. RIght now, the spelling used by you is found 6 times, while the suggested spelling is found 20+ times (and `Baremetal Operator` is found another 20+ times :roll_eyes: )